test/unit/write-app-declarations.test.ts

```diff
@@ -12,7 +12,7 @@ describe('find config', () => {
   beforeEach(async () => {
     await fs.ensureDir(fixtureDir)
   })
-  afterEach(() => fs.remove(declarationFile))
+  afterEach(async () => await fs.remove(declarationFile))
```

Comment:
this is async

Reply:
It already returned a Promise that Jest awaits. This just adds another